### ISSUE TITLE
Update for Angular V15

### ISSUE THREAD

**Issue by yvesguilherme** [sclausen/ngx-mqtt#221]
Hi guys, how are you?

I have a problem with angular in version 15 to use this library, is there any prediction that it can be migrated? I urgently need to use mqtt in a project, but I can't find anything compatible with the current version of angular... 😢 

can anyone help me?

hugs!

**Comment by sclausen** [comment]
I‘m currently on vacation but have a few spare days left after our return and planned to work on several reported issues. Unfortunately that’s in nearly two weeks, so I can‘t give you a quick fix in the next days 😔

**Comment by yvesguilherme** [comment]
Hi @sclausen , how are you?
Don't worry, I want to thank you for contacting me... 
I think I can help with this... but first, I need to finish up some issues in my post-graduation....
Let's talk when you get back from your vacation ok?
Hugs!

**Comment by sclausen** [comment]
Just to keep you updated since I haven’t pushed my progress 🫣
I recreate for angular 15 and am currently working on an example.

**Comment by yvesguilherme** [comment]
Oh @sclausen, not problem haha, thank you so much...

**Comment by goncalocaraujo** [comment]
> 


Is there any expected release date?

**Comment by sclausen** [comment]
No, not at all. This project is not at the top of the list of things I want to do in my free time.

**Comment by sclausen** [comment]
I just published an alpha of v15. Without the examples I told you I worked on to not let you wait any longer.

**Comment by yvesguilherme** [comment]
Thank you...
